nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/NebulaOptions.scala

```diff
@@ -142,7 +142,11 @@ class NebulaOptions(@transient val parameters: CaseInsensitiveMap[String])(
   }
 
   def getReturnCols: List[String] = {
-    returnCols.split(",").toList
+    if (returnCols.trim.isEmpty) {
```

Comment:
if `returnCols` is empty why don't report an error？

Reply:
Empty returnCols means return all properties, keep consistent with client and storage interface.